chore(curriculum): add transcripts for styling forms module (#58443)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/lecture-best-practices-for-styling-forms/672bc9d9eecb65cdf63491de.md

> Watch the lecture video and answer the questions below.

Reviewer: ```suggestion
Watch the video or read the transcript and answer the questions below.
```

# --transcript--

> When Should You Use `appearance: none` to deal with issues styling search inputs and checkboxes?

Reviewer 2: ```suggestion
When should you use `appearance: none` to deal with issues styling search inputs and checkboxes?
```

Let's learn when to use `appearance: none` to address styling issues for search input, checkboxes and radio buttons.

> Browsers apply default styling to a lot of elemets, in the case of input elements your ability to style them with CSS can feel rather restricted. So, you may want to use `appearance: none` to hide aspects of the default element and build your own instead. For example, this would hide the default checkboxes for a checkbox input allowing you to use custom indicators like a green tick and a red X to show the state. For a search input, webkit based browsers will show a default search icon and a cancel button. Hiding these allows you to create your own indicators that would appear on all browsers. 

Reviewer 3: ```suggestion
Browsers apply default styling to a lot of elements. In the case of input elements your ability to style them with CSS can feel rather restricted. So, you may want to use `appearance: none` to hide aspects of the default element and build your own instead. For example, this would hide the default checkboxes for a checkbox input allowing you to use custom indicators like a green tick and a red X to show the state. For a search input, WebKit based browsers will show a default search icon and a cancel button. Hiding these allows you to create your own indicators that would appear on all browsers. 
```

> Webkit is a software engine that helps  web browsers display websites. Browsers like Safari use webkit to make sure web pages look and work correctly. This `appearance: none` CSS property gives you complete control over the styling, but it does come with some things to look out for. the default interactive components of input elements include features such as focus and error indicators which you will need to ensure aren't lost.

Reviewer 3: ```suggestion
WebKit is a software engine that helps web browsers display websites. Browsers like Safari use WebKit to make sure web pages look and work correctly. This `appearance: none` CSS property gives you complete control over the styling, but it does come with some things to look out for. The default interactive components of input elements include features such as focus and error indicators which you will need to ensure aren't lost.
```
